src/Satooshi/Bundle/CoverallsV1Bundle/Api/Jobs.php

```diff
@@ -186,7 +186,5 @@ public function getJsonFile()
         if (isset($this->jsonFile)) {
```

Comment:
I'm fine with that, but for it please also update that properties are nullable in their docblocks